README.md

```diff
@@ -69,9 +69,9 @@ tfrecorder create-tfrecords \
 
 Using Python interpreter:
 ```python
-import tfrecorder
+from tfrecorder import check
```

Comment:
Thanks for this @jmarrietar! 

@cfezequiel do we want to add check into __init__.py for update line 74 below to include tfrecorder.check.check_tfrecords()?

Reply:
Thanks for catching this @jmarrietar. We appreciate your contribution. I do think it may be better to add the `check` import to `__init__.py` for consistency with the other operations (e.g. `client`, `accessor`) so that `import tfrecord` would be sufficient to access them, as @mbernico suggested. But `from tfrecorder import check` would still be valid.